modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/itemActions.spec.ts

```diff
@@ -47,111 +56,663 @@ test.afterEach(async ({dataSetManagerApiHelpers}) => {
 });
 
 test(
-	'There is a message if there are no item actions',
+	'Check interactive options in item action form',
```

Comment:
I squashed couple of tests for performance, as they all check features that are only in frontend. Tests that save, edit, delete, or interact are in separate tests. Frontend-only features include:
- available select options
- localizeable inputs
- tooltips
- icons filtering 

Note - localizeable inputs checks were frontend-only also in Poshi. We already have full localization check in [visualizationModes.spec.ts](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/visualizationModes.spec.ts#L889).